### PR TITLE
use envvar instead of simple var

### PR DESCRIPTION
**Pull request by Hoshiyo**
cf: https://github.com/CanalTP/fab_kirin/pull/7

### CODE REVIEW

File: kirin/default_settings.py [hunk 1]
```diff
@@ -1,23 +1,27 @@
+# encoding: utf-8
+
+import os
+
 #URI for postgresql
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 #http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
-SQLALCHEMY_DATABASE_URI = 'postgresql://navitia:navitia@localhost/kirin'
+SQLALCHEMY_DATABASE_URI = os.getenv('JORMUNGANDR_SQLALCHEMY_DATABASE_URI', 'postgresql://navitia:navitia@localhost/jormungandr')
```
Comment (kinnou02) [inline kirin/default_settings.py]: except that we are in kirin, not jormungandr

File: kirin/default_settings.py [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
-NAVITIA_URL = 'https://api.navitia.io/'
+NAVITIA_URL = os.getenv('JORMUNGANDR_NAVITIA_URL', 'https://api.navitia.io/')
 
 NAVITIA_TIMEOUT = 5
 
-NAVITIA_INSTANCE = 'sncf'
+NAVITIA_INSTANCE = os.getenv('JORMUNGANDR_NAVITIA_INSTANCE', 'sncf')
 
-NAVITIA_TOKEN = None
+NAVITIA_TOKEN = os.getenv('JORMUNGANDR_NAVITIA_TOKEN', 'None')
```
Comment (kinnou02) [inline kirin/default_settings.py]: None shouldn't be a string

File: kirin/default_settings.py [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
 
-CONTRIBUTOR = 'realtime.ire'
+CONTRIBUTOR = os.getenv('JORMUNGANDR_CONTRIBUTOR', 'realtime.ire')
 
 
-DEBUG = True
+DEBUG = bool(os.getenv('DEBUG', False))
```
Comment (kinnou02) [inline kirin/default_settings.py]: you must use something like `flask_restful.inputs.boolean`
`bool` will convert anything to `True`

File: kirin/default_settings.py [hunk 4]
```diff
@@ -1,23 +1,27 @@
...
 
 #rabbitmq connections string: http://kombu.readthedocs.org/en/latest/userguide/connections.html#urls
-RABBITMQ_CONNECTION_STRING = 'pyamqp://guest:guest@localhost:5672//?heartbeat=60'
+RABBITMQ_CONNECTION_STRING = os.getenv('RABBITMQ_CONNECTION_STRING', 'pyamqp://guest:guest@localhost:5672//?heartbeat=60')
```
Comment (kinnou02) [inline kirin/default_settings.py]: you should prefix everything by `KIRIN_`

File: kirin/default_settings.py [hunk 5]
```diff
@@ -1,23 +1,28 @@
...
+from flask_restful.inputs import boolean
+
 #URI for postgresql
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 #http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
-SQLALCHEMY_DATABASE_URI = 'postgresql://navitia:navitia@localhost/kirin'
+SQLALCHEMY_DATABASE_URI = os.getenv('JORMUNGANDR_SQLALCHEMY_DATABASE_URI', 'postgresql://navitia:navitia@localhost/kirin')
 
-NAVITIA_URL = 'https://api.navitia.io/'
+NAVITIA_URL = os.getenv('JORMUNGANDR_NAVITIA_URL', 'https://api.navitia.io/')
```
Comment (antoine-de) [inline kirin/default_settings.py]: isn't it the same for this (s/JORMUNGANDR/KIRIN/) ?

**Pull request merged**